result/isislsp.yaml

```diff
@@ -367,8 +383,21 @@ components:
             down:
               x-field-uid: 2
         prefix_attributes:
-          $ref: '#/components/schemas/IsisLsp.PrefixAttributes'
+          description: |-
+            Extended Prefix Attribute flags container sub-TLV is type 4.
+          type: array
```

Comment:
Why is this changed ? Is this related to SR ? Is this not backward compatibility breaking ?